README.md

```diff
@@ -2,7 +2,7 @@
 
 [XWiki Platform](https://www.xwiki.org/xwiki/bin/view/Documentation/) is a generic wiki platform offering runtime services for applications built on top of it.
 
-XWiki Commons, XWiki Rendering and XWiki Platform are part of the [XWiki.org](http://www.xwiki.org/) software forge. They are released together and share the same version.
+XWiki Commons, XWiki Rendering, and XWiki Platform are part of the [XWiki.org](http://www.xwiki.org/) software forge. They are released together and share the same version.
```

Comment:
Why is this a grammar mistake? IIRC both `XWiki Commons, XWiki Rendering and XWiki Platform ` and `XWiki Commons, XWiki Rendering, and XWiki Platform ` are correct. I would rather consider that the comma is not needed in a list, before `and`.

Reply:
@vmassol: @acotiuga is right. The comma before AND is **optional** and really not needed in this case. See https://www.grammarly.com/blog/comma-before-and/ .

Reply:
Thanks @mflorea I had actually checked it and found this link too but all I could extract from it is that both are allowed. We can revert if you prefer, I have no problem with that.